ruby/interpreter/RubyEval.hpp

```diff
@@ -6,8 +6,8 @@
 #ifndef RUBYEVAL_included
 #define RUBYEVAL_included
 
-#include "ruby.h"
-#include "./RubyException.hpp"
+#include <ruby.h>
+#include "RubyException.hpp"
```

Comment:
Just curios , why are using relative path above but keeps the hpp name here?

Reply:
I'm not sure I understand the question, could you rephrase please?